api/defs/wrapAlignment.schema.json

```
@@ -0,0 +1,16 @@
+{
+    "$schema": "https://raw.githubusercontent.com/lenra-io/ex_component_schema/beta/priv/static/draft-lenra.json",
+    "$id": "alignment.schema.json",
```

```suggestion
    "$id": "wrapAlignment.schema.json",
```
Vérifier que ça n'affecte pas d'autres fichiers